azure-mgmt-compute/src/main/java/com/microsoft/azure/management/compute/VirtualMachineSize.java

```diff
@@ -5,32 +5,32 @@
  */
 public interface VirtualMachineSize {
     /**
-     * Gets the VM size name.
+     * @return the VM size name
      */
     String name();
 
     /**
-     * Gets the Number of cores supported by a VM size.
+     * @return the Number of cores supported by a VM size
      */
     Integer numberOfCores();
 
     /**
-     * Gets the OS disk size allowed by a VM size.
```

Comment:
It's helpful to provide a general one-line javadoc on top of the `@return` line for getters.